src/macros.js

```diff
@@ -3,8 +3,11 @@
  * This can be used to define some commands in terms of others.
  */
 
+import symbols from "./symbols";
+import utils from "./utils";
+
 // This function might one day accept additional argument and do more things.
-function defineMacro(name, body) {
+function defineMacro(name: string, body: string | () => string) {
```

Comment:
Cool! Mind adding `@flow` at the top of this file?

Reply:
Oops, indeed!  Clearly still a flow noob...

Reply:
Me too, for all that's worth. ;)